docs/userguide.md

```diff
@@ -930,3 +932,21 @@ only one instance is responsible for reconciliation. You can refer to the
 [helm chart configuration](https://github.com/kubernetes-sigs/kubefed/tree/master/charts/kubefed#configuration)
 to configure parameters for leader election to tune for your environment
 (the defaults should be sane for most environments).
+
+## Limitations
+### Immutable Fields
+Federation API does not implement immutable fields in the federated resource yet.
+
+A kubernetes resource field can be patched during runtime to change the resource
+specification. But the immutable field is not allowed to be modified after the
+resource was created.
+
+For federated resource, `spec.template` defines target resource specification.
+You can change any fields in runtime. Federation API does not check attribute of
```

Comment:
Replace the text up to 'For example' with: 

Though it is possible to modify any template field of a federated resource (or set an override for the field), changing the value of an immutable field will prevent all subsequent updates from completing successfully.  This will be indicated by a propagation status of `UpdateFailed` for affected clusters.  These errors can only be resolved by reverting the template field back to the value set at creation.